config/config.prod.yaml

```diff
@@ -93,6 +93,8 @@
   bugzilla_user_id: 430528
   description: Firefox Android Team Tag
   parameters:
+    jira_components:
+      use_bug_component_with_product_prefix: true
```

Comment:
@onemahon do you only want the `product::component` format added to the each issue? If so, you'll also want to also set `use_bug_component: false` here. Otherwise, each issue will have two components set: the `component` format _and_ the `product::component` format.

Reply:
Ooooo interesting! While that could arguably be useful, that wasn't my intention - thanks for the heads up. Fixed!

Would you prefer me to squash that into the first commit, or can that be done in Github when this PR gets merged?

Reply:
We'll squash + merge, so you're all good!